src/app/app.component.ts

```diff
@@ -30,6 +30,7 @@ export class AppComponent implements OnInit {
 
   private autoCollapse = 992;
 
+  public darkModeEnabled: boolean = false
```

Comment:
Missing a semicolon:

```suggestion
  public darkModeEnabled: boolean = false;
```